dobi.yaml

```diff
@@ -0,0 +1,93 @@
+
+meta:
```

Comment:
OK, I must be missing something. Suppose I'm running on arch, the fabfile fails, I decide to go docker as you suggest. I want to build the docker image myself (instead of using an official image.) That image is ubuntu-based, so the fabfile using apt-get works within the image. dobi does not contribute to installing anything to that scenario, am I not correct?